src/journal/migrations/0010_auto_20170514_1426.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-05-14 14:26
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('journal', '0009_auto_20170428_2204'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Cover',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('cover_image', models.ImageField(blank=True, upload_to=b'cover_image_uploads/')),
```

Comment:
There is a typo here:
`upload_to=b'cover_image_uploads/'`
Should be:
upload_to='cover_image_uploads/'